WordPress/Jetpack/Classes/NUX/JetpackPrologueViewController.swift

```diff
@@ -12,6 +13,12 @@ class JetpackPrologueViewController: UIViewController {
         return view
     }()
 
+    private lazy var motion: CMMotionManager = {
```

Comment:
👋 @salimbraksa - would this mean it gets initialized every time? The main reason I went for `lazy` was so that it only got used once it was needed, and only with the new landing screen.

However, I noticed that the call on `viewWillDisappear` would initialize it, so perhaps that call should be behind the feature flag check as well. Do you have thoughts on that? 🙇

Reply:
Ah I see, and yes you are right, `viewWillDisappear` would initialize it and `angleForDeviceOrientation` too. Having the call behind the feature flag would work, but what do you think of making the `motion` property optional?

```swift
private lazy var motion: CMMotionManager? = {
    guard FeatureFlag.newLandingScreen.enabled else {
        return nil
    }
    let manager = CMMotionManager()
    manager.deviceMotionUpdateInterval = Self.Constants.deviceMotionUpdateInterval
    return manager
}()
```
And in this case, maybe we could use `let` instead of `lazy var`.

Reply:
That's a good thought @salimbraksa, thanks! I may go this route, but also want to evaluate whether or not we should make the motion manager more of a global singleton per Apple's guidelines ([more context here](https://github.com/wordpress-mobile/WordPress-iOS/pull/19434#discussion_r993968026)).

Since there are only two known spots where we use sensor data (here and the About Screen), I feel that the risk is overall low.

Reply:
Addressed in 09254a88500410aa15912a5f4b1f0d7484c834ed @salimbraksa.

Reply:
@twstokes This looks good to me, and I think the global singleton approach can be done in another PR since it involves refactoring the About screen too.

Reply:
Great! Sounds good. 🙇